### PR TITLE
Added location User Trace

### PR DESCRIPTION
**Pull request by adnanjt**

### CODE REVIEW

File: app/views/DR/ReportScreen/ReportScreenQuestions.js [hunk 1]
```diff
@@ -199,6 +201,7 @@ export default function ReportScreenQuestions({ navigation }) {
           onPress={async () => {
             if (isLastStep) {
               const { covidId } = await sendDataToApi();
+              SetStoreData(COVID_ID, covidId);
               console.log('ELCOVID', covidId);
```
Comment (iocedano) [inline app/views/DR/ReportScreen/ReportScreenQuestions.js]: RIP! please

File: app/views/LocationTracking.js [hunk 2]
```diff
@@ -216,6 +217,36 @@ class LocationTracking extends Component {
...
+            longitude: location.longitude,
+            time: location.time,
+            covidId: covidId,
+          };
+
+          fetch('https://webapps.mepyd.gob.do/contact_tracing/api/UserTrace', {
+            method: 'POST',
+            headers: { 'Content-Type': 'application/json' },
+            body: JSON.stringify(data.body),
+          })
```
Comment (iocedano) [inline app/views/LocationTracking.js]:
```suggestion
          const body = JSON.stringify({
            latitude: location.latitude,
            longitude: location.longitude,
            time: location.time,
            covidId: covidId,
          });

          fetch('https://webapps.mepyd.gob.do/contact_tracing/api/UserTrace', {
            method: 'POST',
            headers: { 'Content-Type': 'application/json' },
            body,
          })
```

File: app/views/LocationTracking.js [hunk 3]
```diff
@@ -216,6 +217,36 @@ class LocationTracking extends Component {
...
+            .then(function(response) {
+              return response.json();
+            })
+            .then(data => {
+              console.log('[INFO] ' + 'Location API: ' + JSON.stringify(data));
+              return data;
+            })
+            .catch(error => {
+              console.error('[ERROR] ' + error);
```
Comment (iocedano) [inline app/views/LocationTracking.js:243]: can the User see the error?

File: app/views/DR/ReportScreen/ReportScreenQuestions.js (hunk 1 above)
Comment (RutsSantos) [inline app/views/DR/ReportScreen/ReportScreenQuestions.js]: Remove console log

File: app/views/LocationTracking.js [hunk 4]
```diff
@@ -216,6 +217,36 @@ class LocationTracking extends Component {
...
+          fetch('https://webapps.mepyd.gob.do/contact_tracing/api/UserTrace', {
+            method: 'POST',
+            headers: { 'Content-Type': 'application/json' },
+            body: JSON.stringify(data.body),
+          })
+            .then(function(response) {
+              return response.json();
+            })
+            .then(data => {
+              console.log('[INFO] ' + 'Location API: ' + JSON.stringify(data));
```
Comment (RutsSantos) [inline app/views/LocationTracking.js]: Is this console.log necessary?

**Review by iocedano — CHANGES_REQUESTED** [review]
LGTM!
Just worry about the console logs the may show to the Users

**Review by RutsSantos — COMMENTED** [review]
LGTM, but solve the comments

File: app/views/LocationTracking.js (hunk 3 above)
Comment (adnanjt) [reply app/views/LocationTracking.js:243]: no

**Review by RutsSantos — APPROVED** [review]

**Review by JoseFcoEstrella — APPROVED** [review]
LGTM

File: app/views/LocationTracking.js (hunk 2 above)
Comment (iocedano) [reply app/views/LocationTracking.js]: Unnecessary variable declaration `data`.

We could keep it simple by declaring `const body`

**Pull request merged**